src/odemis/model/_components.py

```diff
@@ -247,6 +247,15 @@ def __setstate__(self, state):
         _vattributes.load_vigilant_attributes(self, vas)
         _dataflow.load_events(self, events)
 
+    def __setattr__(self, name, value):
+        # Detect that the user is trying to replace a VigilantAttribute, which is
+        # most likely a typo of forgetting VA.value .
+        # On a Component, it's fishy, but on a Proxy, there is just no reason to
+        # do that.
+        if hasVA(self, name):
+            raise AttributeError("Cannot override VigilantAttribute")
```

Comment:
Is it maybe useful here to add the name you're trying to set, in this AttributeError?

Reply:
Indeed. I somehow assumed that from the backtrace it'd be possible to find out which VA is overridden. However, it's really simple and simpler to be more explicit.